### DISCUSSION TITLE
Auto start app

### DISCUSSION THREAD

**Discussion by boskoe** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#395] [Q&A] [answered]
Hi,

New to Macs. I purchased a license and managed to successfully uninstall the free version and get Pro 1.07 installed and I think working. One thing I can't find is how to make AlDente start automagically when I log in. I'd greatly appreciate any advise given.

Thanks,

Doug

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @boskoe,

Thank you for purchasing AlDente Pro.
To activate launch at login you have to open AlDente Pro, go into settings -> General Settings. You will find the switch in the lower left corner.

Cheers,
Matthias

**Reply by boskoe** [reply]
Matthias,

Thank you for prompt answer. Slightly embarrassed that I hadn't seen it hiding in plain sight.

Doug

**Comment by UkeFan** [comment]
Hi there!
With mac os 12.6 it isn't working. Though activated I have to start al dente after each reboot manually.
Idea?!
Thanks!

**Reply by MatthiasKerbl** [reply]
Hi @UkeFan,

That's weird, it should definitely work. Did you try to activate/deactivate the start at login or reinstall AlDente?

Cheers,
Matthias

**Reply by DCalzolari** [reply]
Hello,
same problem here with version 1.27 Pro.
